### ISSUE TITLE
After changing citation key format, obsidian no longer imports note from zotero and it's not fixed even after I restore the format..

### ISSUE THREAD

**Issue by s12baak** [retorquere/zotero-better-bibtex#2901]
### Debug log ID

8A4LB7XV-apse/6.7.203-7

### What happened?

Thank you for all your contribution to solving problems with zotero better bibtex. 
It was well used with obsidian-zotero integration until I made a change in citation key formula. The problem might have been preceded before the change, or could be problem with obsidian but I can't tell at the moment. 
All I can explain is, obsidian do create md note when I import from zotero, but displays title only and contents are missed. When I view the file with data explorer, surprisingly it seems that metadatas are retrieved but can't be displayed through template. However, I haven't made any change in obsidian template before the problem started.  

I want to know where I must focus in order to solve the problem. Below is the errors message I am looking in zotero. Please give me a hint what could have caused the problem. Thank you!

Errors:
1718092188783	addons.xpi	WARN	Checking C:\Program Files\Zotero\distribution\extensions for addons
1718092188956	addons.xpi-utils	WARN	addMetadata: Add-on undefined is invalid: Error: Invalid addon ID: expected addon ID undefined, found undefined in manifest(resource://gre/modules/addons/XPIDatabase.jsm:3108:15) JS Stack trace: addMetadata@XPIDatabase.jsm:3108:15
processFileChanges@XPIDatabase.jsm:3569:26
checkForChanges@XPIProvider.jsm:3121:60
startup@XPIProvider.jsm:2532:12
callProvider@AddonManager.sys.mjs:228:31
_startProvider@AddonManager.sys.mjs:536:17
startup@AddonManager.sys.mjs:742:14
startup@AddonManager.sys.mjs:3666:26
observe@amManager.sys.mjs:73:29
1718092189454	addons.webextension.better-bibtex@iris-advies.com	WARN	Loading extension 'better-bibtex@iris-advies.com': Reading manifest: Using experimental APIs requires a privileged add-on.
1718092189454	addons.webextension.daeda@mit.edu	WARN	Loading extension 'daeda@mit.edu': Reading manifest: Using experimental APIs requires a privileged add-on.
[JavaScript Error: "Error: Failed to load resource://services-settings/remote-settings.sys.mjs" {file: "resource://gre/modules/PartitioningExceptionListService.sys.mjs" line: 88}]

**Comment by retorquere** [comment]
Diagnosis will have to start on the Obsidian end. I don't know why it's not being displayed.

**Comment by s12baak** [comment]
Oh, I see. It means I should try debugging from obsidian, right? I was confused where to look for the errors since I believed the problem started as I changed the citation key formula... I'll try to look up errors from obsidian first, thanks!

**Comment by retorquere** [comment]
Any news from the obsidian integration dev?

**Comment by s12baak** [comment]
Sorry for late reply. I was busy with other work lately so I didn't make
inquiry to zotero integration yet. It takes time for me to question in
english, especially when I want to explain in details of what happened and
what I have tried since I'm not good with english.. As I returned to my own
work recently, I'm to inquire zotero integration github. I will let you
know if there is any reply or diagnosis made. Thank you for your kindness
and concern! Have a good day :)

2024년 6월 23일 (일) 오후 9:32, Emiliano Heyns ***@***.***>님이 작성:
